cmds/ls/ls.go

```diff
@@ -17,8 +17,8 @@
 package main
 
 import (
-	"flag"
 	"fmt"
+	flag "github.com/spf13/pflag"
```

Comment:
super nit! please separate this from std lib imports

```go
import (
  "fmt"

   flag "github.com..."
)
```

Reply:
done. Now you need to do a fast review :-)